packages/jitm/src/class-pre-connection-jitm.php

```diff
@@ -108,6 +118,26 @@ private function filter_messages( $message_path ) {
 		return $formatted_messages;
 	}
 
+	/**
+	 * Returns true when one of the plugins in the message's required active plugin list
+	 * is active.
+	 *
+	 * @param array $message The message array.
+	 *
+	 * @return bool True when a matching plugin is active, else false.
+	 */
+	private function is_matching_active_plugin( $message ) {
```

Comment:
We take a bit of a different approach in the `Jetpack` method:
https://github.com/Automattic/jetpack/blob/e940e90c3f26a6f24f8e7fccd72a1dd6360be2b5/class.jetpack.php#L2085-L2093

I think that's to handle network-activated plugins on Multisite as well? Maybe we should do the same here?

Reply:
Core's `is_plugin_active()` function also [checks for network activated plugins](https://core.trac.wordpress.org/browser/tags/5.5.1/src/wp-admin/includes/plugin.php#L531), so core's `is_plugin_active()` function and `Jetpack::is_plugin_active()` will return the same results.

However, I just realized that the pre-connection JITMs won't be displayed when the plugin is installed as an mu plugin, since mu plugins aren't aren't considered 'active'. So, I need to rethink this. Maybe the plugins should set up their own pre-connection JITMs via a filter.